docs/sharing/design-integrations.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Design integrations'
+---
+
+Storybook supports first-class integrations with well-known design tools, speeding up the development cycle by enabling you to cross-reference designs with your existing stories.
+
+![Storybook design integrations](./storybook-design-integrations.jpeg)
+
+For development teams, Storybook provides the necessary tools to build, [test](../writing-tests/introduction.md), and [document](../writing-docs/introduction.md) UI components. [Integrating](https://storybook.js.org/addons/tag/design/) your Storybook with popular design tools (for example, [Figma](https://www.figma.com/), [Zeplin](https://zeplin.io/)) unlocks new workflows for you and your team. You prototype the UI in a design tool, build and test it in Storybook, allowing you to surface issues and ensure consistent and reliable UIs.
+
+## Figma
+
+[Figma](https://www.figma.com/) is a popular collaborative UI design tool used by [Slack](https://slack.com/), [Twitter](https://twitter.com/), and [Dropbox](https://www.dropbox.com/) teams. It allows multiple people to work on the same design simultaneously in the browser.
+
+If you're working with Figma, you have two options for connecting your Storybook. You can use the [Storybook addon designs](https://storybook.js.org/addons/storybook-addon-designs), which allows you to preview your designs alongside the currently selected story.
+
+![Storybook addon figma](./storybook-figma-addon.png)
+
+Or via a plugin developed by the Storybook maintainers. [Storybook Connect](https://www.figma.com/community/plugin/1056265616080331589/Storybook-Connect) is the official Figma plugin that lets you link stories to your Figma components. You can preview your existing stories in the Figma design workspace when configured. Below is an overview of how to get started with the plugin.
```

Comment:
Simplified the wording and cut out the bit about it being an official plugin. IMO if its going in the docs then its official by default. 

```suggestion
Or use [Storybook Connect](https://www.figma.com/community/plugin/1056265616080331589/Storybook-Connect) to embed stories into your Figma design workspace. Below is an overview of how to get started with the plugin.
```

Reply:
Update with the latest commit. Thanks for the suggestion @winkerVSbecks